python/test/t_HSICStat_std.py

```diff
@@ -48,7 +48,9 @@
     test = X.getMarginal(i)
     # Set input covariance scale
     Cov1.setScale(test.computeStandardDeviation())
-    hsicIndex = estimatorTypeV.computeHSICIndex(test, Y, Cov1, Cov2, W)
+    CovMat1 = Cov1.discretize(test)
```

Comment:
```suggestion
    CovMat1 = Cov1.discretize(test)
```
In Python too variable names should start with lowercase letters.